client/jetpack-cloud/sections/onboarding-tours/add-new-site-tour-step-2/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslate } from 'i18n-calypso';
+import { useContext } from 'react';
+import GuidedTour from 'calypso/jetpack-cloud/components/guided-tour';
+import { useSelector } from 'calypso/state';
+import { getPreference } from 'calypso/state/preferences/selectors';
+import SitesOverviewContext from '../../agency-dashboard/sites-overview/context';
+import { SiteData } from '../../agency-dashboard/sites-overview/types';
+import { JETPACK_MANAGE_ONBOARDING_TOURS_PREFERENCE_NAME } from '../constants';
+
+interface Props {
+	siteItems: Array< SiteData >;
+}
+
+export default function AddNewSiteTourStep2( { siteItems }: Props ) {
+	const translate = useTranslate();
+	const { mostRecentConnectedSite } = useContext( SitesOverviewContext );
+	const hasFinishedStep1 = useSelector( ( state ) =>
+		getPreference( state, JETPACK_MANAGE_ONBOARDING_TOURS_PREFERENCE_NAME[ 'addSiteStep1' ] )
+	);
+	// We should only render the second step if the first one has finished (hasFinishedStep1) and we have a new connected site (mostRecentConnectedSite).
+	const shouldRenderAddSiteTourStep2 = hasFinishedStep1 && mostRecentConnectedSite;
+
+	const tourHTMLTarget =
+		siteItems.length > 20
```

Comment:
This should be `<=20`, or the latter two operands should be flipped.

Reply:
Just < would be better. Just spotted the last commit changed this to >. Siteitems will only ever be a max of 20, even if there are a lot more sites (because of pagination). So to ensure we have less than one page, we can really only check for 19 or lower to be safe.